### PR TITLE
Add slide notes to pptx

### PR DESCRIPTION
**Pull request by seawatts**
This adds the ability to gather the notes on the slides. 

Added a new argument `include_slide_notes` to `partition_pptx.` This defaults to `False` to stay backwards compatible.

PPTX API Reference https://python-pptx.readthedocs.io/en/latest/user/notes.html

### CODE REVIEW

**Review by MthwRobinson — COMMENTED** [review]
@seawatts - Thanks for the PR! Could you add an entry to the `CHANGELOG`? Otherwise LGTM!

**Comment by seawatts** [comment]
Hi @MthwRobinson, thanks for taking a look at the PR! Should I add it under `0.8.2-dev1` - `Features` section?

**Comment by MthwRobinson** [comment]
@seawatts - Yep features section is good!

**Comment by seawatts** [comment]
Great! @MthwRobinson updated!

**Review by MthwRobinson — APPROVED** [review]
LGTM!

File: unstructured/partition/pptx.py [hunk 1]
```diff
@@ -70,6 +73,13 @@ def partition_pptx(
     for i, slide in enumerate(presentation.slides):
         metadata = ElementMetadata.from_dict(metadata.to_dict())
         metadata.page_number = i + 1
+        if include_slide_notes and slide.has_notes_slide == True:
```
[the remark this answers is not in the document]
Comment (MthwRobinson) [reply unstructured/partition/pptx.py:76]: Few other linting errors, I think it wants `slide.has_notes_slide is True` and `if notes_slide.notes_text_frame is not None`.

**Review by MthwRobinson — APPROVED** [review]

File: unstructured/partition/pptx.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (seawatts) [reply unstructured/partition/pptx.py]: Awesome, just updated. Thanks

**Comment by seawatts** [comment]
@MthwRobinson looks like I'm getting the error in the lint check

```
ruff . --select I,UP015,UP032,UP034,UP018,COM,C4,PT,SIM,PLR0402 --ignore PT011,PT012,SIM117
black --line-length 100 unstructured --check
make: black: No such file or directory
make: *** [Makefile:178: check-src] Error 127
Error: Process completed with exit code 2.
```

**Comment by MthwRobinson** [comment]
@seawatts - Yeah I think that's CI being finicky and not an actual failure. I'll turn it off and turn it back on again 😄

**Comment by seawatts** [comment]
@MthwRobinson haha, classic, thanks!

**Pull request merged**